contracts/passport/verifiers2/noir/registerIdentity_2_256_3_6_264_336_20_2448_5_2008.sol

```diff
@@ -0,0 +1,3416 @@
+            ) // vk.TABLE4.y
+            mstore(
+                add(_vk, 0x500),
+                0x2d40d36c04e79e82b3bd10a48e9dd49f65b63438a59e9084e4638a0c5b828ee7
+            ) // vk.TABLE_TYPE.x
+            mstore(
+                add(_vk, 0x520),
+                0x231c6ece43d94893a240e9ce1b17e9928215bd903cf180583f117c8ed6199a0b
+            ) // vk.TABLE_TYPE.y
+            mstore(
+                add(_vk, 0x540),
+                0x0ccbccea07dce330a3dd765af8fef64b475d48b010b8949d24e023fc9fe8b1a1
+            ) // vk.ID1.x
+            mstore(
+                add(_vk, 0x560),
+                0x113d2561615333d5937982babd2cd289a954bd436b6347079c906d509dd9b8e2
+            ) // vk.ID1.y
+            mstore(
+                add(_vk, 0x580),
+                0x0ed55598187451f5a2e942f23b05e57086a463791370828c86b748cbdb822625
+            ) // vk.ID2.x
+            mstore(
+                add(_vk, 0x5a0),
+                0x1fb8be30690548ca18ecf12b230464414787ceeb64dcbe10c091c1c401b5dc49
+            ) // vk.ID2.y
+            mstore(
+                add(_vk, 0x5c0),
+                0x27571788c3332317d5cd1ec64c02b1242ca8deaac58317530d329fd84f53edb1
+            ) // vk.ID3.x
+            mstore(
+                add(_vk, 0x5e0),
+                0x044ff0cd5ccac95ec85b1b3763439e4991d2e39f90ffc7a9d5e4878f4cda30bd
+            ) // vk.ID3.y
+            mstore(
+                add(_vk, 0x600),
+                0x29469741811e18fbd457910ea950f8a223e15f12294b366f764cbc0c8672f645
+            ) // vk.ID4.x
+            mstore(
+                add(_vk, 0x620),
+                0x1b9bbc5d17824730e9ed6548e5e215acb9e1c22e356834e7252d6eb7a8afbb21
+            ) // vk.ID4.y
+            mstore(add(_vk, 0x640), 0x00) // vk.contains_pairing_point_accumulator
+            mstore(add(_vk, 0x660), 0) // vk.pairing_point_accumulator_public_input_indices
+            mstore(
+                add(_vk, 0x680),
+                0x260e01b251f6f1c7e7ff4e580791dee8ea51d87a358e038b4efe30fac09383c1
+            ) // vk.g2_x.X.c1
+            mstore(
+                add(_vk, 0x6a0),
+                0x0118c4d5b837bcc2bc89b5b398b5974e9f5944073b32078b7e231fec938883b0
+            ) // vk.g2_x.X.c0
+            mstore(
+                add(_vk, 0x6c0),
+                0x04fc6369f7110fe3d25156c1bb9a72859cf2a04641f99ba4ee413c80da6a5fe4
+            ) // vk.g2_x.Y.c1
+            mstore(
+                add(_vk, 0x6e0),
+                0x22febda3c0c0632a56475b4214e5615e11e6dd3f96e6cea2854a87d4dacc5e55
+            ) // vk.g2_x.Y.c0
+            mstore(
+                _omegaInverseLoc,
+                0x036853f083780e87f8d7c71d111119c57dbe118c22d5ad707a82317466c5174c
+            ) // vk.work_root_inverse
+        }
+    }
+}
+
+/**
+ * @title Ultra Plonk proof verification contract
+ * @dev Top level Plonk proof verification contract, which allows Plonk proof to be verified
+ */
+abstract contract BaseUltraVerifier {
```

Comment:
Won't this abstract contract naming fuck us up?

Reply:
It is auto generated(

Reply:
I mean if we have two verifiers, won't the solidity compiler complain?

Reply:
It should not because of fully qualified paths